ext/spl/tests/fixedarray_003.phpt

```diff
@@ -1,86 +0,0 @@
---TEST--
-SPL: FixedArray: Iterators
---FILE--
-<?php
-class A extends SplFixedArray {
```

Comment:
Replace this with a test that overrides getIterator(), based on a generator for example?

Reply:
Done.